wrappers/MathCAD/Prime/README.rst

```diff
@@ -1,19 +1,45 @@
-CoolProp for MathCAD Prime
-==========================
+CoolProp Wrapper for MathCAD Prime 3.x or later
+==================================================
 
-Info
-----
+| Copyright Scott Polak and Ian Bell, 2013
+| Updated by Jeff Henning, 2016
 
-Wrapper built by Scott Polak (scott.polak@aei.com), December 2013
+Prerequisites
+==============
 
+* You will need to have Visual Studio 2008 installed.  Alternatively newer versions of Microsoft Visual Studio C++ should be fine.  Unfortunately, you will need the professional version of Visual Studio C++ as Mathcad Prime libraries are 64-bit and require the 64-bit compiler; the Express version of Visual Studio will not work.
 
-To Use/Build
-------------
+* You will need CMake version 2.8.12 or later from https://cmake.org/download/
 
-* Compiled for MathCAD Prime 3.0 64-bit version - the .lib file is for 64-bit, if you want a 32-bit version you need access to the 32-bit version of the mcaduser.lib file for Prime 3.0
+* You will need to install Git-SCM for Windows.  You can install this from https://git-for-windows.github.io
 
-* The mcaduser.lib and mcadincl.h files from MathCAD need to be included and linked when building the �CoolProp 4.0 Beta Mathcad Prime 3.0 Wrapper.cpp� file. 
+* You will need Anaconda/Miniconda Python, which you can get from https://store.continuum.io/cshop/anaconda
+	
 
-* If you have Mathcad Prime 3.0 installed, you need to put the DLL in the folder: C:\Program Files\PTC\Mathcad Prime 3.0\Custom Functions , or wherever your MathCAD is installed
+To Build
+========
+
+* **Go to the top level CoolProp directory and make a build directory** (something like \buildprime)::
```

Comment:
git clone --recursive https://github.com/coolprop/coolprop should do it

On Fri, May 6, 2016 at 7:37 PM, Jeff Henning <notifications@github.com>
wrote:

> In wrappers/MathCAD/Prime/README.rst
> <https://github.com/CoolProp/CoolProp/pull/1047#discussion_r62409015>:
>
> >
> > -* If you have Mathcad Prime 3.0 installed, you need to put the DLL in the folder: C:\Program Files\PTC\Mathcad Prime 3.0\Custom Functions , or wherever your MathCAD is installed
> > +To Build
> > +========
> > +
> > +* **Go to the top level CoolProp directory and make a build directory** (something like \buildprime)::
>
> You're going to test my Git knowledge... I don't even remember how I did
> this the first time. I'll peruse Chacon's book and resuscitate some brain
> cells.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/CoolProp/CoolProp/pull/1047/files/9e94bb2fceae09e7a6b21b50851e78fb806b6626#r62409015>
>